package.json

```diff
@@ -23,19 +23,15 @@
     "build:parsers": "nx run parsers:build",
     "build:vscode-react": "nx run vs-code-react:build",
     "build:vscode-designer": "nx run vs-code-designer:build && nx run vs-code-data-mapper-react:build && nx run vs-code-designer-react:build && nx run vs-code-react:build",
-    "build:vscode-datamapper": "nx run vs-code-data-mapper:build && nx run vs-code-data-mapper-react:build",
+    "build:vscode-datamapper": "nx run vs-code-designer:build && nx run vs-code-data-mapper-react:build",
     "build:utils": "nx run utils:build",
     "build:docs": "nx build logic-apps-ux-docs",
     "start:docs": "nx serve logic-apps-ux-docs",
     "bump": "standard-version --no-verify",
     "vscode:designer:pack:step1": "cd ./dist/apps/vs-code-designer && npm install",
     "vscode:designer:pack:step2": "node extension-copy-svgs.js designer",
```

Comment:
NIT: We can remove the `designer` word since we are not using it any more in the `extension-copy-svgs.js` (this is passed as the `argunmentsArray` )

Reply:
Got it, so just run the `nx run vs-code-data-mapper-react:build` for the vscode-datamapper. Checked it is working as expected and pushed.

Reply:
ohhh ok if that works fine its great. But I meant in the `vscode:designer:pack:step2` script (line 32) sorry if I caused misunderstanding.

Reply:
Modified to remove "designer" argument from step 2 as per your explanation!